README.md

```diff
@@ -136,6 +153,39 @@ public function boot()
 }
 ```
 
+Alternatively, you can now simply tag your collector in the container and it will be discovered:
+
+```php
+// app/Providers/AppServiceProvider.php
+
+use YourApp\Collectors\MailMessageCollector;
+
+public function boot()
+{
+    $this->app->tag(MailMessageCollector::class, \AG\ElasticApmLaravel\ServiceProvider::COLLECTOR_TAG);
+}
+```
+
+## Distributed Tracing
+
+[Distributed tracing](https://www.elastic.co/guide/en/apm/get-started/current/distributed-tracing.html) allows you to associate transactions in your Laravel application with transactions in services your application consumes. For example, if your Laravel application calls REST resources as part of handling a request, the REST transaction details will appear within your application transaction in Elastic APM. You enable distributed tracing by including an appropriate header in the http request your application makes to another service. For example:
+
+```php
+$transaction = $agent->currentTransaction();
```

Comment:
Should we create an Agent facade? Otherwise user will need to get the instance like `app(Agent::class);` every time they need access to the Agent.

Reply:
I've added an `ApmAgent` facade and exposed two methods for helping with the distributed tracing header.

Reply:
Great, thanks!